rmc/utils/constraint.py

```diff
@@ -534,6 +534,9 @@ def create_constraint_prep_ht(
     ht = get_per_variant_expected_dataset(
         directory_post_fix=directory_post_fix, path_post_fix=path_post_fix
     ).ht()
+    # filter to Canonical Ensembl transcripts
```

Comment:
```suggestion
    # Filter to Canonical Ensembl transcripts
    # NOTE: All MANE Select transcripts in GENCODE v39 are canonical, but not all canonical transcripts are MANE Select
```
(black may reformat this if it's too long)